pkg/driverbuilder/builder/builders.go

```diff
@@ -106,6 +113,132 @@ func Script(b Builder, c Config, kr kernelrelease.KernelRelease) (string, error)
 	return buf.String(), nil
 }
 
+type GCCVersionRequestor interface {
+	GCCVersion(kr kernelrelease.KernelRelease) float64
+}
+
+type Image struct {
+	GCCVersion []float64
+}
+
+var images = map[string]Image{
+	"buster": {
+		GCCVersion: []float64{4.8, 5, 6, 8},
+	},
+	"bullseye": {
+		GCCVersion: []float64{9, 10},
+	},
+	"bookworm": {
+		GCCVersion: []float64{11, 12},
+	},
+}
+
+func defaultGCC(kr kernelrelease.KernelRelease) float64 {
```

Comment:
Nit: might want a test for this function

Reply:
Agree!